src/analyze/analyze-verify.c

```diff
@@ -11,10 +11,23 @@
 #include "manager.h"
 #include "pager.h"
 #include "path-util.h"
+#include "string-table.h"
 #include "strv.h"
 #include "unit-name.h"
 #include "unit-serialize.h"
 
+static int log_syntax_callback(const char *unit, int level, void *log_syntax_callback_userdata) {
+        Set **s = log_syntax_callback_userdata;
```

Comment:
Oh, I see that this was changed from `userdata` to `log_syntax_callback_userdata` following my comment… I only had the global variable in mind. Of course this local variable can keep the short name. It's not a big issue, but maybe if there's another round, rename it back here in the function.

Reply:
yeah, please name the parameter just "userdata" when it has local scope. long identifiers are useful in public symbols, but for local variables we shouldn't get lost in verbosity